src/components/InputRadio/InputRadio.tsx

```diff
@@ -110,7 +110,7 @@ const InputRadio = forwardRef(
     }, [isGroupStyle, addonProps]);
 
     return (
-      <div className="bsds-field">
+      <div className="bsds-field bsds-mt-2x">
```

Comment:
thoughts on this vs input-checkbox.scss

Reply:
Both solutions work, just came up with them at different times in different contexts. Which feels cleaner?

Reply:
i don't have a strong preference, just as long as we're consistent across similar components